### PR TITLE
Ohm 737 release openhim core

### PR DESCRIPTION
**Pull request by BMartinos**
Bump package version to reflect the version the OpenHIM core is on
Add a CHANGELOG file to track changes made between version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+* Fix channel filtering for non-admins 
+* Handle responses without timestamps
+* Updated a metrics check that verifies the timestamp value to check that its a object (Date) instead of a number
+  * Replace the verifification mechanism to be instanceof Date instead of typeof
+* Add a shebang character to the url of the transaction link 
+  * Add missing ! to links
+* update the 'update' object which is checked when creating metrics so that metrics are created with mediators
+* Include time in filter dates
+* Determine user access level using channel permission groups (non-admin user can retireve transaction body when requesting multiple transactions - reporting as no admin user)
```
Comment (Zooloo2014) [inline CHANGELOG.md]: Spelling here - should be 'retrieve'

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Code Cleanup
+
+* Don't set secureProtocol - already supported in supported versions of Node.js
+* Remove the harmony flag for start up
+* Remove the load testing code run by Artillery.
```
Comment (Zooloo2014) [inline CHANGELOG.md]: For consistency, can you remove the full stop here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* Remove un-used packages
+* Remove the declaration of the Log model schema as this is only used within the tests
+* Remove package script command that isnt being used.
```
Comment (Zooloo2014) [inline CHANGELOG.md]: Full stop - please remove this one too

**Comment by BMartinos** [comment]
Thanks @Zooloo2014 I have addressed your comments and review for another review

**Pull request merged**